### DISCUSSION TITLE
0.1.0

### DISCUSSION THREAD

**Discussion by michele-perrone** [michele-perrone/OpenPiano#2] [Announcements]
# About this release
This is the first binary release of Open Piano. It is targeted at Windows x64, macOS x64/M1 (universal binary), and GNU/Linux x64. In this release, only the standalone binaries are provided (no plugins). There is no installer: you only need to download and unzip the correct folder, then run Open Piano.

# Important notes
Please note that:
* This is still alpha-quality software, so don't expect to record you next hit with it (and it's not really a piano yet).
* Open Piano is quite CPU intensive. If you hear crackling, consider using a higher buffer size.
* Changing the sample rate affects the quality of the string simulation: a higher sampling rate corresponds to a higher number of spatial samples for each string. You will hear the difference especially for the higher notes.

# Description of release contents
Inside each archive, you will find two executables/apps:
* `Open Piano`: this is the base version of Open Piano. The GUI features only a keyboard.
* `Open Piano - Spectral Analysis`: this version is built from the `spectral-analysis` branch. It features a real-time STFT and harmonic ratio visualization (you can switch between the two). Issues/bugs:
   * There is a known bug: if you change the sample rate/buffer size in the plugin audio settings, the visualization stops, and you have to restart the plugin.
   * The CPU usage of the Spectral Analysis version is higher than the normal one. It is intended less for playing and more for exploring how the spectrum changes at different hammer velocities.

<hr /><em>This discussion was created from the release <a href='https://github.com/michele-perrone/OpenPiano/releases/tag/0.1.0'>0.1.0</a>.</em>

**Comment by HiImCris** [comment]
This is an awesome project. Keep it up!

**Reply by michele-perrone** [reply]
Hello HilmCris, there will soon be a new release with better multithreading and less audio glitches, stay tuned.
